tools/coqdoc/cdglobals.mli

```diff
@@ -39,6 +39,7 @@ val title : string ref
 val externals : bool ref
 val coqlib : string ref
 val coqlib_path : string ref
+val coqcorelib_path : string ref
```

Comment:
Is this going to be used?

Reply:
Good catch, this was introduced in a clenaup commit that has been squashed now. Will fix.